### ISSUE TITLE
Checklist disappears when switching view then checking checkbox

### ISSUE THREAD

**Issue by Nyco** [nextcloud/deck#2566]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

**Describe the bug**
Checklists and description (text content) of a card simply disappears when I try to toggle view and then check a checkbox in checklist

**To Reproduce** (not completely sure yet)
1. Create card
2. Edit description adding some ```- [ ] blah blah``` lines
3. Close editor
4. Switch view: modal to sidebar or sidebar to modal
5. Check one checkbox
=> The text content disappears

**Expected behavior**

**Screenshots**

1. Create card, edit checklist, in sidebar
![Capture d’écran de 2020-11-16 22-33-53](https://user-images.githubusercontent.com/25348/99310983-1e933c00-285c-11eb-82d1-f064463aa6cb.png)
2. Save checklist (close editing)
![Capture d’écran de 2020-11-16 22-34-05](https://user-images.githubusercontent.com/25348/99310991-22bf5980-285c-11eb-931b-ac45eca11716.png)
3. Switch to modal view
![Capture d’écran de 2020-11-16 22-34-15](https://user-images.githubusercontent.com/25348/99311001-2652e080-285c-11eb-9025-3393b64356ea.png)
4. Check on "zwei" entry, description content disappears
![Capture d’écran de 2020-11-16 22-34-30](https://user-images.githubusercontent.com/25348/99311007-2a7efe00-285c-11eb-97cf-4d216640897b.png)

**Client details:**
 - OS: [e.g. iOS] Ubuntu 20.04 LTS
 - Browser [e.g. chrome, safari] Chrome latest Version 86.0.4240.198 (Build officiel) (64 bits)
 - Device: [e.g. iPhone6, desktop] Laptop Lenovo X1 Carbon

<details>
<summary>Server details</summary>

**Operating system**:

**Web server:**

**Database:**

**PHP version:**

**Nextcloud version:** (see Nextcloud admin page) 19.0.4, Deck 1.2.0

**Where did you install Nextcloud from:**

**Signing status:**

```
Login as admin user into your Nextcloud and access
http://example.com/index.php/settings/integrity/failed
paste the results here.
```

**List of activated apps:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```

**Nextcloud configuration:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder

or

Insert your config.php content here
Make sure to remove all sensitive content such as passwords. (e.g. database password, passwordsalt, secret, smtp password, …)
```

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

</details>

<details>
<summary>Logs</summary>

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```

</details>

**Comment by reox** [comment]
duplicate of #2351?

**Comment by stefan-niedermann** [comment]
Yep, let's track it down in #2351 and close this.

**Comment by ajeoc** [comment]
I have the same issue like Nyco and it is according to the description in #2351 and #2061 not a duplicate.
When i have a card with markdown like:
- [ ] 1
- [ ] 2
- [ ] 3
and i close and reopen the card details. i check one box and all is gone.

**Comment by juliushaertl** [comment]
Duplicate of #2580, fixed with #2584
